packages/std/src/addresses.rs

```diff
@@ -93,25 +166,25 @@ impl PartialEq<Addr> for String {
 
 impl From<Addr> for String {
     fn from(addr: Addr) -> Self {
-        addr.0
+        addr.as_str().to_owned()
```

Comment:
Can we also call this `to_string`? Sometimes it is nice to have explicit names (`as_str`, `to_string`) and not just `from` and `into`

Reply:
`to_string` is already implemented via the `Display` trait implementation. This implies a `ToString` implementation.